libcxx/CMakeLists.txt

```diff
@@ -504,11 +504,10 @@ remove_flags(-Wno-pedantic -pedantic-errors -pedantic)
 # Required flags ==============================================================
 function(cxx_add_basic_build_flags target)
 
-  # Require C++20 for all targets. C++17 is needed to use aligned allocation
-  # in the dylib. C++20 is needed to use char8_t.
+  # Use C++23 for all targets.
   set_target_properties(${target} PROPERTIES
-    CXX_STANDARD 20
-    CXX_STANDARD_REQUIRED YES
+    CXX_STANDARD 23
+    CXX_STANDARD_REQUIRED OFF
```

Comment:
I guess this is to avoid breaking the docs bot?

Reply:
Yes exactly.

Reply:
Can you add comment why it's OFF, maybe even a TODO.

Reply:
f97a579b74a6e65a3f58813b36f6a789102ce315